sonic-chassisd/scripts/chassisd

```diff
@@ -395,10 +423,20 @@ class ModuleUpdater(logger.Logger):
                 current_midplane_state = fvs[CHASSIS_MIDPLANE_INFO_ACCESS_FIELD]
 
             if midplane_access is False and current_midplane_state == 'True':
-                self.log_warning("Module {} lost midplane connectivity".format(module_key))
+                if self.is_module_reboot_expected(module_key):
+                    self.module_reboot_set_time(module_key)
+                    self.log_warning("Expected: Module {} lost midplane connectivity.".format(module_key))
+                else:
+                    self.log_warning("Unexpected: Module {} lost midplane connectivity.".format(module_key))
             elif midplane_access is True and current_midplane_state == 'False':
                 self.log_notice("Module {} midplane connectivity is up".format(module_key))
-
+                # clean up the reboot_info_table
+                if self.module_reboot_table.get(module_key) is not None:
+                    self.module_reboot_table._del(module_key)
+            elif midplane_access is False and current_midplane_state == 'False':
+                if self.is_module_reboot_system_up_expired(module_key):
+                    self.log_warning("Unexpected: Module {} lost midplane connectivity.".format(module_key))
```

Comment:
Shouldn't the syslog say that module midplane connectivity was not restored in the timeout period.
The loss of midplane connectivity was already logged when the transition from Up to Down was noticed.